packages/manager/modules/pci/src/projects/project/training/jobs/info/translations/Messages_fr_FR.json

```diff
@@ -29,5 +29,9 @@
   "pci_projects_project_training_jobs_info_title_6": "Infos",
   "pci_projects_project_training_jobs_info_title_actions": "Actions",
   "pci_projects_project_training_job_logs_text_pre_running": "Le job est encore en attente de démarrage… L'onglet se rafraîchira automatiquement une fois que le job aura commencé.",
-  "pci_projects_project_training_jobs_resubmit": "Relancer"
+  "pci_projects_project_training_jobs_resubmit": "Relancer",
+
```

Comment:
**nitpick:** Can you remove this empty line?
```suggestion
```

Reply:
Sure, done here: https://github.com/ovh/manager/pull/8578/commits/5dba8a2bed58e82a1c550049e847285f3031e864